docs.rs doc build broken

I noticed that https://docs.rs/async-executor/1.13.0 has a broken build. The docs.rs build logs say:

```rust
[INFO] [stderr] error[E0658]: `#[doc(cfg)]` is experimental
[INFO] [stderr]   --> src/lib.rs:61:20
[INFO] [stderr]    |
[INFO] [stderr] 61 | #[cfg_attr(docsrs, doc(cfg(any(feature = "static"))))]
[INFO] [stderr]    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[INFO] [stderr]    |
[INFO] [stderr]    = note: see issue #43781 <https://github.com/rust-lang/rust/issues/43781> for more information
[INFO] [stderr]    = help: add `#![feature(doc_cfg)]` to the crate attributes to enable
```

It appears that the problem is that #122 added a use of `#[doc(cfg(...))]` without also adding `#![feature(doc_cfg)]`.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -37,7 +37,7 @@
 #![doc(
     html_logo_url = "https://raw.githubusercontent.com/smol-rs/smol/master/assets/images/logo_fullsize_transparent.png"
 )]
-#![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 
 use std::fmt;
 use std::marker::PhantomData;

